packages/components/atoms/f-button/test-utils/component-objects/f-button.component.js

```diff
@@ -1,10 +1,22 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
+const { buildUrl } = require('@justeat/f-wdio-utils/src/storybook-extensions');
```

Comment:
Rather than import `buildUrl` throughout all of the component objects you could add it to the base `page` object class and expose it from there?

Reply:
Yes, exactly, that's the next commit I did.

Here's the list in order:
- https://github.com/dbernar1/fozzie-components/commit/2426f6cad6291aae8d7e3040e3b2a0a6e86a3151
- https://github.com/dbernar1/fozzie-components/commit/6e21160337f5efef608372f571417d9677436c74
- https://github.com/dbernar1/fozzie-components/commit/281d77162e0e39caba2e8ad368fd330a45a4c314
- https://github.com/dbernar1/fozzie-components/commit/0d706629c2862f020c2ec1464c31414b3cb847dd
- https://github.com/dbernar1/fozzie-components/commit/d8c494330334e48ee00203b20fb0c514e2f4a175

Let me know if you prefer that we put them together in one PR instead of separate. That would currently look like this: https://github.com/justeat/fozzie-components/compare/master...dbernar1:remove-component-waitForComponent?expand=1

Thanks!

Reply:
Awesome 👍🏻